lib/licensee/content_helper.rb

```diff
@@ -4,17 +4,87 @@
 module Licensee
   module ContentHelper
     DIGEST = Digest::SHA1
+    START_REGEX = /\A\s*/
```

Comment:
Just curious, why do this and the next two remain outside the hash?

Reply:
Missed a few. Fixed. Good catch,